app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.java

```diff
@@ -34,7 +34,8 @@
 import org.mozilla.telemetry.storage.TelemetryStorage;
 
 public final class TelemetryWrapper {
-    private static final String TELEMETRY_APP_NAME = "Focus";
+    private static final String TELEMETRY_APP_NAME_FOCUS = "Focus";
```

Comment:
We have a dependency on the server side processing here - So I guess it's safer to keep them separate.